inc/template-tags.php

```diff
@@ -45,6 +45,24 @@ function alcatraz_posted_on() {
 }
 
 /**
+  * Prints the page title unless hide tile is checked.
+  */
+function alcatraz_entry_title() {
+
+	$hide_title = get_post_meta( get_the_ID(), '_alcatraz_hide_title', true );
+
+	if ( $hide_title ) {
+		return;
+	}
+
+	echo '<header class="entry-header">';
+
+	echo the_title( '<h1 class="entry-title">', '</h1>' );
+
+	echo '</header>';
+}
```

Comment:
If you do something like this, then we'll be able to modify the entry titles easier:

```php
function alcatraz_entry_title() {
    $custom_title = apply_filters( 'alcatraz_entry_title', '' );
    if ( ! empty( $custom_title ) ) {
        echo $custom_title;
        return;
    }
    ...
```
This would let us use a different wrap element or heading level or add content before or after the titles pretty easily.